hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1639,6 +1639,18 @@ def recommission_regionserver(server_name_string, encoded_region_names)
       @admin.recommissionRegionServer(server_name, region_names_in_bytes)
     end
 
+    #----------------------------------------------------------------------------------------------
+    # Retrieve latest balancer decisions made by LoadBalancers
+    def get_balancer_decisions
+      balancer_decisions_responses = @admin.getBalancerDecisions
+      balancer_decisions_resp_arr = []
+      balancer_decisions_responses.each { |balancer_dec_resp|
+        balancer_decisions_resp_arr << balancer_dec_resp.toJsonPrettyPrint
+      }
+      puts 'Retrieved BalancerDecision Responses from HMaster'
+      puts balancer_decisions_resp_arr
```

Comment:
Only return the array here, this may be used for programmatic things. Put the part that prints the result into the command impl which invokes this admin.rb function.